astroquery/esa/neocc/core.py

```diff
@@ -0,0 +1,453 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""
+Main module from ESA NEOCC library. This module contains the two main
+methods of the library: *query_list* and *query_object*. The information
+is obtained from ESA Near-Earth Object Coordination Centre's (NEOCC) web
+portal: https://neo.ssa.esa.int/.
+"""
+
+import time
+
+from astroquery.query import BaseQuery
+from astroquery.utils import async_to_sync
+
+from astroquery.esa.neocc import lists, tabs
+
+__all__ = ['neocc', 'ESAneoccClass']
+
+
+@async_to_sync
+class ESAneoccClass(BaseQuery):
```

Comment:
the classname looks weird, why not use NEOCCClass (we only have ESAHubbleClass as a distinguished as there was already hubble querying functionality via MAST when the ESA module was assed).

Or, of course, it's totally fine to keep ESA in the name, but the capitalization should still be fixed, I'm thinking of ``ESANeoccClass``, or ``ESANEOCCClass``?

Reply:
Done